tests/check-meet-csv

```diff
@@ -107,6 +107,7 @@ KnownFederations = {
     "IPL",
     "IrishPF",  # Irish Powerlifting Federation (IPF affiliate).
     "IrishPO",  # Irish Powerlifting Organization (WPU/IPL affiliate).
+    "KPF",  # Icelandic IPF Affiliate.
```

Comment:
The federation's name should be `KRAFT` per https://www.facebook.com/KRAFTIS/, with `meet-data/kraft/`.

Reply:
Ok- I wasn't sure what name they went by since they list themselves as KPF on meet pages.